packages/firebase_ui_auth/lib/src/views/email_link_sign_in_view.dart

```diff
@@ -83,6 +83,14 @@ class _EmailLinkSignInViewState extends State<EmailLinkSignInView> {
               ),
             ],
             const SizedBox(height: 8),
+            UniversalButton(
```

Comment:
If this is used as a root view, `pop()` will close the app. We could have this configurable via widget property (e.g. `showBackButton` or use `Navigator.canPop`

Reply:
hm, I think most of the views that have this button without any conditions are indeed not supposed to be used as a root view, but if you think some could – feel free to make a similar change.

Reply:
Only displaying this button if Navigator.canPop is true. Confirmed with my local app that if I push() a route to the EmailLinkSignInScreen then the "Go Back" button is shown, but if I pushReplacement() the same route (making it the root view) then "Go Back" is not shown.